game/theater/conflicttheater.py

```diff
@@ -481,7 +483,7 @@ def add_preset_locations(self) -> None:
         for static in self.helipads:
             closest, distance = self.objective_info(static)
             closest.helipads.append(
-                PointWithHeading.from_point(
+                Helipad.from_point(
```

Comment:
Are helipads only allowed at FOBs? This doesn't appear to be enforced, but they only affect the parking capacity for FOBs.

That's probably fine? It might confuse some people but since we can't actually account for the differences in parking types currently that's probably best. People can add helipads to airports if they want and they'll behave as a guaranteed unsheltered parking space. If they affected the capacity we'd have no way to stop people (even the AI) from filling those helipads with B-52s.

Alternatively, just check for it here and raise an error if that's not something we want to account for just yet.

Reply:
Yes, the idea was that you can put them on an airfield, and helos at this airfield should then prioritize helipads, so they don't block the runway for dozens of minutes. But i'd say it's not recommended, and up to campaign designer. 

> That's probably fine? It might confuse some people but since we can't actually account for the differences in parking types currently that's probably best. People can add helipads to airports if they want and they'll behave as a guaranteed unsheltered parking space. If they affected the capacity we'd have no way to stop people (even the AI) from filling those helipads with B-52s.

Exactly, this is why i prevented them from affecting capacity on airfields for now.
But this will remain an issue. We should assign a dedicated slot for each aircraft bought, and have a smart system to assign them, and make it so a player can't end up with 4 B-52 on an airfield that can only host 2 large aircraft and 10 fighter-sized ones.

Reply:
Fine with me. It's probably worth a comment on `Airfield.total_aircraft_parking`